TestingGrounds/TestingGrounds/DoublyLinkedList.h

```diff
@@ -142,7 +152,7 @@ class DoublyLinkedList
     uint32_t getMiddle(void)
     {
         if (head == nullptr)
-            return 0;
+            return -1;
```

Comment:
This will implicitly cast to an unsigned int and you will return a _big fat number._

Reply:
Love me some big fat numbers

Reply:
Oh yeah that's big